<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Configuration</class>
 <widget class="QDialog" name="Configuration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>421</width>
    <height>269</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>421</width>
    <height>269</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>421</width>
    <height>269</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Configuration</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBoxConfiguration">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>210</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxASCII_RTU">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>201</width>
     <height>221</height>
    </rect>
   </property>
   <property name="title">
    <string>Modbus ASCII/RTU</string>
   </property>
   <widget class="QComboBox" name="comboBoxDebit">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>60</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentText">
     <string>9600 bit/s</string>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <item>
     <property name="text">
      <string>1200 bit/s</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2400 bit/s</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4800 bit/s</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>9600 bit/s</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>19200 bit/s</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>38400 bit/s</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBoxLongueur">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>100</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <item>
     <property name="text">
      <string>7 bits</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>8 bits</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBoxParite">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <item>
     <property name="text">
      <string>Paire</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Impaire</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pas de parité</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="comboBoxBitStop">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>1 bit</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2 bits</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="labelPortCOM">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelDebit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Débit</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelLongueur">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Longueur</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelParite">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Parité</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelBitStop">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Bit de stop</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxPortCOM">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>101</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>COM1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM6</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM7</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM8</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>COM9</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QComboBox" name="comboBoxProtocole">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>ASCII</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>RTU</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>TCP</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="labelProtocole">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Protocole Modbus</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxTCP">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>191</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Modbus TCP</string>
   </property>
   <widget class="QLineEdit" name="lineEditAdresseIP">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="labelAdresse">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Adresse IP</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButtonAideConfiguration">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>150</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Afficher l'aide / Rafraîchir</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Adresse Afficheur</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBoxAdresseAfficheur">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>127</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonConfigDefaut">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>180</y>
     <width>151</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Configuration par défaut</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBoxConfiguration</sender>
   <signal>accepted()</signal>
   <receiver>Configuration</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBoxConfiguration</sender>
   <signal>rejected()</signal>
   <receiver>Configuration</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
